### PR TITLE
fixing_trade_profile__ui_time_format

### PR DESCRIPTION
**Pull request by faizan2590**

### CODE REVIEW

File: src/d3a/d3a_core/sim_results/endpoint_buffer.py [hunk 1]
```diff
@@ -236,9 +237,12 @@ def _populate_core_stats_and_sim_state(self, area):
             self._populate_core_stats_and_sim_state(child)
 
     def update_stats(self, area, simulation_status, progress_info, sim_state):
+        from d3a_interface.constants_limits import DATE_TIME_UI_FORMAT
```
Comment (hannesdiedrich) [inline src/d3a/d3a_core/sim_results/endpoint_buffer.py:240]: Please move to top.

**Review by spyrostz — DISMISSED** [review]
LGTM

**Review by hannesdiedrich — APPROVED** [review]
LGTM

**Review by spyrostz — APPROVED** [review]
LGTM

**Pull request merged**